src/themes/masschallenge/modules/dimmer.overrides

```diff
@@ -1,3 +1,3 @@
 .ui.page.modals.dimmer.transition.visible.active {
-  background-color: transparent;
+  background-color: rgba(0,0,0,0.5);
```

Comment:
why are we hard coding this?

Reply:
This is meant to change the background of dimmer from `transparent` to `rgba(0,0,0,0.5)`
> Looking back at that and in light of how we're using modals now, I think we should make the background more distinct. Could you update the override to:
background-color: rgba(0,0,0,.5);